src/ChainRulesTestUtils.jl

```diff
@@ -14,6 +14,22 @@ export test_scalar, frule_test, rrule_test, isapprox, generate_well_conditioned_
 Base.isapprox(d_ad::DoesNotExist, d_fd; kwargs...) = error("Tried to differentiate w.r.t. a `DoesNotExist`")
 Base.isapprox(d_ad::AbstractDifferential, d_fd; kwargs...) = isapprox(extern(d_ad), d_fd; kwargs...)
 
+"""
+    _make_fdm_call(fdm, f, ȳ, xs, ignores) -> Tuple
+
+Call `FiniteDifferences.j′vp`, with the option to ignore certain `xs`.
+
+# Arguments
+- `fdm::FiniteDifferenceMethod`: How to numerically differentiate `f`.
```

Comment:
We specify the type of `fdm` but in the actual function signature we do not.

Reply:
It's true that for this function to the work we need it to be this type

But it's not type constrained in the definition because we don't dispatch on it

This docstring is anyway just here to help future contributors who will most likely be replacing this function